cap/modules/schemas/utils.py

```diff
@@ -40,6 +42,34 @@ def _filter_only_latest(schemas_list):
     return [next(g) for k, g in groupby(schemas_list, lambda s: s.name)]
 
 
+def _get_serialized_response(schema, config=None, resolve=None):
```

Comment:
not needed. we only care about `versions`, why serialise everything

Reply:
we get `links` from `SchemaSerializer`. For versions, we don't need `serialize` but for links yes.